rpcs3/Emu/Cell/SPUThread.cpp

```diff
@@ -4458,7 +4458,9 @@ bool spu_thread::set_ch_value(u32 ch, u32 value)
 
 				if (res == CELL_EAGAIN)
 				{
+					// Restore mailboxes state
 					ch_out_mbox.set_value(data);
+					ch_in_mbox.try_pop(data);
```

Comment:
This to me looks like it needs some testing. Any ideas on what games could be affected by this change?

Reply:
EAGAIN is a fake error code for savestates here, it means the syscall ia aborted because it cannot be completed because the  thread-to-be-signalled has aborted execution in savestates.